sphinx_gallery/directives.py

```diff
@@ -125,67 +127,88 @@ def run(self):
         # Retrieve source directory
         src_dir = config.sphinx_gallery_conf["src_dir"]
 
-        # Parse the argument into the individual objects
-        obj_list = []
+        # Parse the argument into the individual args
```

Comment:
`objects` is very general and a bit confusing for me, changed to `args` as I think of them as directive args (maybe 'inputs' is another option?)